src/app/parcels/batch/BatchParcelDataGrid.tsx

```diff
@@ -0,0 +1,267 @@
+"use client";
+
+import { Button } from "@mui/material";
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import React from "react";
```

Comment:
I don't think this has been required for a couple of versions of react now

Reply:
required for testing purposes, Jest throws an error when the test file is run without this line